### PR TITLE
Initial commit adding test suite.

### PR DESCRIPTION
**Pull request by mynamewastaken**
Update Pip* files to add testing packages and add initial test.

Partially addresses issue #69

### CODE REVIEW

File: test_shallow_backup.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import pytest
+
+def test_backup_prompt():
```
Comment (alichtman) [inline test_shallow_backup.py]: This doesn't test any functionality of the program, only that `" "` isn't in a list that's hardcoded and does not contain the string `" "`

This test can never fail... and is therefore useless.

Write a useful test and I'll approve this PR.

One example could be testing the `copying` functionalities with a variety of paths, including edge cases, like a path with escape characters in it.
